widget/view.controller.js

```diff
@@ -415,22 +419,42 @@
                     return connectorService.getConnector(agents[0].conn_name, agents[0].conn_version, agents[0].agent)
                       .then(response => _getConnectorHealth(installedConnector, response, index));
                   } else {
-                    installedConnector.checked = false;
+                    toaster.error({
+                      body: 'The' + installedConnector.label + ' is not configured. To proceed with the wizard, please close the ' + installedConnector.label + ' tab.'
+                    });
                     loadActiveTab(index);
-                    return Promise.resolve(); // Resolve to continue the chain
+                    return Promise.reject(); // Resolve to continue the chain
                   }
                 });
               }
             });
         });
       }, Promise.resolve()); // Start with a resolved promise
-
       promises
         .then(() => {
-          $scope.feedIntegrationTools = _.map(
-            _.filter($scope.installedConnectors, { checked: true }),
-            'label'
-          );
+          $scope.installedConnectors.reduce((promise, configConnector, index) => {
+            return promise.then(() => {
+              const metaData = {
+                "name": $scope.saveSchedules[index].name,
+                "description": "Metadata for " + $scope.saveSchedules[index].description,
+                "modified_by": "3451141c-bac6-467c-8d72-85e0fab569ce",
```

Comment:
Please add comment for this uuid, i.e. to whom it belongs to?

Reply:
this is csadmin user uuid and will add comment in next PR